setup.cfg

```diff
@@ -31,7 +31,7 @@ package_dir =
 packages = find_namespace:
 python_requires = >= 3.8
 install_requires =
-    stactools == 0.4.3
```

Comment:
just a general question -- why are we pinning to a version here instead of twiddle-wacka 0.4.5? @gadomski

Reply:
🤷🏼 **stactools-packages** are right on that line between "library" and "appliaction", IMO. They're rarely if ever used for their API in downstream code, except in a deployed situation (e.g. in an ETL pipeline) where they usually can control their own environment. But yeah, on balance it probably should be a `>=` or `~=` (I generally prefer `>=` for libraries).

Reply:
lol, just hit the need for this